chainerio/containers/zip.py

```diff
@@ -98,7 +98,24 @@ def info(self):
 
     def stat(self, path):
         self._open_zip_file()
-        return self.zip_file_obj.getinfo(path)
+        try:
+            stat = self.zip_file_obj.getinfo(path)
```

Comment:
Maybe appending `/` here if it doesn't finish with `/` would be simpler than handling errors in except clause.

Reply:
We need to check both with `/` (directories) and without `/` (normal files), since we do not know if a path without `/` is a directory or not